kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -23,8 +23,12 @@ use super::runtime_tables;
 /// Number of constraints produced by the argument.
 pub const CONSTRAINTS: u32 = 7;
 
-/// The number of random values to append to columns for zero-knowledge.
-pub const ZK_ROWS: usize = 3;
+/// The number of points that are evaluated in the protocol (zeta and zeta * omega).
+pub const EVALS: u64 = 2;
+
+/// The number of rows that are added to a witness/circuit in order to provide zero-knowledge.
+/// We might actually only need `EVALS` but let's be double safe.
+pub const ZK_ROWS: u64 = EVALS + 1;
```

Comment:
as said elsewhere, these should be moved to a constants.rs file

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/91350343175bf874d16ada51a0ee486a6135db73